src/locales/it/settings.ts

```diff
@@ -98,5 +98,10 @@ export const settings: SimpleTranslationEntries = {
   "gamepadSupport": "Supporto Gamepad",
   "showBgmBar": "Mostra Nomi Musica",
   "moveTouchControls": "Move Touch Controls",
-  "shopOverlayOpacity": "Opacità Finestra Negozio"
+  "shopOverlayOpacity": "Opacità Finestra Negozio",
+  "shopCursorTarget": "Shop Cursor Target",
+  "items": "Items",
+  "reroll": "Reroll",
+  "shop": "Shop",
+  "checkTeam": "Check Team"
```

Comment:
Is it better like this @torranx ?
```suggestion
  "shopCursorTarget": "Target Cursore Negozio",
  "items": "Oggetti",
  "reroll": "Rerolla",
  "shop": "Negozio",
  "checkTeam": "Squadra"
```